daphne_worker/src/dap.rs

```diff
@@ -173,9 +111,11 @@ impl<'srv> HpkeDecrypter for DaphneWorker<'srv> {
     ) -> std::result::Result<bool, DapError> {
         let version = self.try_get_task_config(task_id).await?.as_ref().version;
         Ok(self
-            .get_hpke_receiver_config(HpkeReceiverKvKey {
-                version,
-                hpke_config_id: config_id,
+            .get_hpke_receiver_config(version, |config_set| {
+                config_set.get(&config_id).map(|_| {
```

Comment:
this would result in a an `Option<bool>` and the `is_some` check below would be wrong. I'd have to change it to `.unwrap_or(false)` which is one of my personal pet peeves because of how unreadable it is :sweat_smile: